_posts/2015-10-14-Four-days-DDD-course.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+name: When you get more than what you were expecting
+title: When you get more than what you were expecting
+date: 2015-10-14 09:35:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/when-you-get-more-than-what-you-were-expecting/
+image:
+    src: /assets/img/custom/blog/DDD_SkillsMatter.jpg
+tags:
+- DDD
+- Learning
+-
+---
+Almost 2 weeks ago, I attended a course at [SkillsMatter](http://skillsmatter.com/) about DDD. The first 2 days were intensive and a reinforce of a lot of concepts that I already had and learning a few more.
+
+I really loved some analogies that [Alberto](https://twitter.com/ziobrando) showed us: like the one of the top gun game where he explain to us that our domain is a plain and that when you want to make a shot you have to make the sight aim to the target. That’s what our model should do. And if you can reduce that delta, then you have the perfect model for your domain.
+
+But what make me really recommend you to take this course was the last 2 days, where [Alberto](https://twitter.com/ziobrando) was phenomenal making as see the importance of making a context map and understanding all the external systems that interact with your system and essentially understanding the role of each one off then.
+
+And then there was the last day: [Alberto](https://twitter.com/ziobrando) told us we would watch some videos.(Isn’t that boring?!?!) And it was in the last day, you know! We were really tired because you had been bombarded 3 days with a lot of information that you would like to retain, but your brain can’t take it anymore. And you are going to see a video?!? I thought I would just fall asleep… Wasn’t I wrong!!! The videos were about a software engineer in a meeting with a business domain expert. So enlightening! I remember some meetings that I had before, I recognize some of the behaviors in both sides. And I hope I realized how I can improve my attitude in a future meeting, so I’ll be more focus in understanding the business logic instead of trying to give then the view that I have of the business.
+
+And then we had, as we say in Portugal, the cherry on the top of the cake: [Alberto](https://twitter.com/ziobrando) made an Event Storming. WOW! It was amazing. Not only participating in it, but been aware of what you should look for in this sessions if you are facilitating one.
```

Comment:
I am not sure what you mean by 'Alberto made an Event Storming'. Do you mean some sort of application? Might be worth defining event storming also.

I don't understand this sentence sorry ...
Not only participating in it, but been aware of what you should look for in this sessions if you are facilitating one.